cob_navigation_global/CMakeLists.txt

```diff
@@ -15,30 +13,19 @@ install(DIRECTORY launch config
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
 
-# rostest for robots
-SET(robotlist
-    cob4-2
-#    cob4-3
-#    cob4-4
-#    cob4-6
-    raw3-1
-    raw3-3
-    raw3-5
-)
-
-SET(envlist
-    ipa-kitchen
-    ipa-apartment
-)
+#############
+## Testing ##
+#############
 
-### TESTING ###
 if(CATKIN_ENABLE_TESTING)
-    # loop through list of robots for testing
-    foreach(robot ${robotlist})
-        foreach(env ${envlist})
-            MESSAGE("testing for robot: ${robot} in env: ${env}")
-            roslaunch_add_file_check(launch ROBOT=${robot} ROBOT_ENV=${env})     # testing environment variables
-            roslaunch_add_file_check(launch robot:=${robot} robot_env:=${env})   # testing launch file arguments
-        endforeach(env)
-    endforeach(robot)
+  find_package(cob_supported_robots REQUIRED)
+  find_package(cob_default_env_config REQUIRED)
+  find_package(roslaunch REQUIRED)
+  foreach(robot ${cob_supported_robots_ROBOTLIST})
+    foreach(env ${cob_default_env_config_ENVLIST})
```

Comment:
hmm, probably because of those lists? Can't you access them this way if you find them via catkin? But then, what about `roslaunch`?